src/gmail.js

```diff
@@ -358,9 +358,9 @@ var Gmail = function(localJQuery) {
 
 
     api.get.storage_info = function() {
-        var div = $(".md.mj").find("div")[0];
-        var used = $(div).find("span")[0].text;
-        var total = $(div).find("span")[1].text;
+        var div = document.querySelectorAll(".md.mj")[0].querySelectorAll("div")[0];
```

Comment:
Iirc querySelector() will select first node only, and is effectively the same as querySelectorAll()[0].